packages/ember-test-app/app/templates/marketing.hbs

```diff
@@ -1,181 +1,206 @@
 {{page-title "MarketingTest"}}
-
-<main data-theme="marketing" class="container p-3">
-  <h4>Billing & Contact Information</h4>
-  <hr class="my-4" />
-  <form class="needs-validation" novalidate="">
-    <div class="row g-3">
-      <div class="col-sm-4">
-        <label for="firstName" class="form-label">
-          First name
-          <span class="text-body-secondary">*</span>
-        </label>
-        <input
-          type="text"
-          class="form-control"
-          id="firstName"
-          placeholder="First"
-          value=""
-          required
-        />
-        <div class="invalid-feedback">
-          Valid first name is required.
+<main data-theme="marketing">
+  <Mktg::Header>
+    <:brand>
+      <img src="https://imageplaceholder.net/50" alt="Icon" />
+    </:brand>
+    <:title>
+        <p class="m-0">Title</p>
+    </:title>
+    <:nav>
+      <Button
+        class="btn-outline-light me-1 rounded-pill"
+      >Prev</Button>
+      <Button
+        class="btn-outline-light me-1 rounded-pill"
+      >Next</Button>
+    </:nav>
+    <:options>
+      <p class="my-0 me-2 fw-bold">Fiber:</p>
+      <LinkTo @route="marketing" class="link-light me-2">The Gig</LinkTo>
+      <p class="my-0 me-2 fw-bold">TV:</p>
+      <LinkTo @route="marketing" class="link-light me-2">Add</LinkTo>
+      <p class="my-0 me-2 fw-bold">Phone:</p>
+      <LinkTo @route="marketing" class="link-light me-2">Add</LinkTo>
+    </:options>
+  </Mktg::Header>
+  <div class="container p-3">
+    <h4>Billing & Contact Information</h4>
```

Comment:
```suggestion
    <h4>Billing &amp; Contact Information</h4>
```

Reply:
I fixed this in the mktg card container PR. Once that one gets approved, I can rebase and this will be resolved, or I can correct it here as well